spec/models/submission_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+RSpec.describe Submission, type: :model do
+  describe "validations" do
+    it { should validate_presence_of(:description) }
+    it { should validate_presence_of(:state) }
+  end
+
+  describe "associations" do
+    it { should belong_to(:fund).class_name("Activity") }
+    it { should belong_to(:organisation) }
+  end
+
+  it "does not allow an association to an Activity that is not level = fund" do
+    programme = create(:programme_activity)
+    submission = build(:submission, fund: programme)
+    expect(submission.valid?).to be false
```

Comment:
We've also used `expect(submission).not_to be_valid` elsewhere (I think `expect(submission).to be_invalid` might also work)